src/main/java/teammates/ui/controller/CreateImageUploadUrlAction.java

```diff
@@ -0,0 +1,40 @@
+package teammates.ui.controller;
+
+import com.google.appengine.api.blobstore.BlobstoreFailureException;
+
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.util.Const;
+import teammates.common.util.GoogleCloudStorageHelper;
+
+public class CreateImageUploadUrlAction extends Action {
```

Comment:
I've added simple javadoc descriptions, but honestly they don't give much more information than class name for the moment ;)